app/src/main/java/fr/free/nrw/commons/profile/achievements/AchievementsFragment.java

```diff
@@ -510,6 +526,19 @@ private void launchAlert(String title, String message){
                 .create()
                 .show();
     }
+    // Launch Alert with a READ MORE button and clicking it open a custom webpage
```

Comment:
Use Multiline JavaDoc `/** * */`

Reply:
+1 to what @4D17Y4 wrote, plus one new line to separate from the previous method.